### PR TITLE
Full update of LB WB

### PR DESCRIPTION
**Pull request by erichrt**
## PR Checklist
Full update of workbook to improve the UX
Adding Health, Data throughput, Flow Distribution and Overview tabs
Adding useful docs and guides for each tab
Renaming charts and updating UI

Screenshots:
image

image

image

image

image

image

image

image

image

image

* [ X] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [X ] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [X ] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ N/A] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

File: Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook [hunk 1]
```diff
@@ -126,7 +124,7 @@
             "version": "KqlParameterItem/1.0",
             "name": "FrontendPublicIPsConstraint",
             "type": 1,
-            "value": "where false",
+            "value": "where id in~ ('/subscriptions/6bb4a28a-db84-4e8a-b1dc-fabf7bd9f0b2/resourcegroups/errobin/providers/microsoft.network/publicipaddresses/myip','/subscriptions/6bb4a28a-db84-4e8a-b1dc-fabf7bd9f0b2/resourcegroups/errobin/providers/microsoft.network/publicipaddresses/ip2')",
```
Comment (sayghosh) [inline Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook:127]: Why this hardcoding?

Comment (erichrt) [reply Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook]: I'm not sure. I expected this to be removed when I set the Load Balancer id to come from query but it automatically populated this parameter with a default value. I've removed this and changed it back to false. I've tested and confirmed that the behavior is as expected with this field returned to false.

File: Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook [hunk 2]
```diff
@@ -5062,42 +5530,49 @@
...
-      ],
       "conditionalVisibility": {
         "parameterName": "Help",
         "comparison": "isEqualTo",
-        "value": "5"
+        "value": "6"
       },
       "customWidth": "50",
-      "name": "LB Backend health probe Fronend IP"
+      "name": "metric - 34 - Copy"
```
Comment (sayghosh) [inline Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook:5568]: @erichrt Please consider having meaningful name against these

File: Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook (hunk 1 above)
Comment (sayghosh) [reply Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook]: @deveshjagwani  can you check how a false hardcoding works? If so, why we need a variable?

Comment (deveshjagwani) [reply Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook]: mailed the details

**Pull request merged**